src/features/query-bar/components/TemplateButton.js

```diff
@@ -53,12 +53,12 @@ class TemplateButton extends React.Component<Props, State> {
         let text;
 
         const isLoadingTemplates = !templates;
-        const hasTemplates = templates && templates.length === 0;
+        const hasNoTemplates = templates && templates.length === 0;
```

Comment:
Negatives are discouraged.

Please revise to "if (!hasTemplates)" and make appropriate fixes.